### PR TITLE
add ToR for TT-OS

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

File: et-tt/tt-os.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+== Task Team on Open Source
+
+=== Terms of Reference
+
+[loweralpha]
+. Define specific roles and responsibilities for INFCOM, in consulation with SERCOM and WMO Secretariat in managing and guiding open-source initiatives.
```
Comment (golfvert) [inline et-tt/tt-os.adoc:6]: In close collaboration with SERCOM, define specific roles and responsibilities of INFCOM and SERCOM in managing and guiding open-source initiatives.

File: et-tt/tt-os.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+== Task Team on Open Source
+
+=== Terms of Reference
+
+[loweralpha]
+. Define specific roles and responsibilities for INFCOM, in consulation with SERCOM and WMO Secretariat in managing and guiding open-source initiatives. 
+. Establish guidelines for the integration of open-source projects in alignment with WMO's technical regulations and standards. 
+. Propose mechanisms for ensuring that open-source development aligns with and accelerates the implementation of key WMO initiatives, particularly Early Warnings for All.
+. Identify and develop and manage a risk register to identify any associated risks with using open-source for WMO Members.
```
Comment (golfvert) [inline et-tt/tt-os.adoc:9]: by WMO Members ?

File: et-tt/tt-os.adoc [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+. Develop strategies and establish guidelines to support Members' digital transformation efforts through open-source solutions. 
+. Outline processes for continuous feedback and improvement in WMO's open-source ecosystem. 
+. Coordinate closely with SERCOM throughout the development of the framework to ensure alignment and complementarity of efforts.
```
Comment (golfvert) [inline et-tt/tt-os.adoc:12]: Move to first or second TOR

**Comment by tomkralidis** [comment]
Thanks for the comments @golfvert.  Updated for review/comment.

**Review by golfvert — APPROVED** [review]

**Comment by tomkralidis** [comment]
@golfvert small update in d2cfbc1b544bf4d27c2ed3d1012d90d83ad6e05e for your review.

**Review by golfvert — APPROVED** [review]

**Pull request merged**